ttnn/cpp/ttnn/tensor/tensor.hpp

```diff
@@ -323,7 +323,7 @@ struct Tensor {
     //                                      Extra Helper Functions
     // ======================================================================================
     StorageType storage_type() const;
-    const tt::tt_metal::LegacyShape strides() const;
+    const ttnn::SimpleShape strides() const;
```

Comment:
not that your PR should fix this, but just wanted to point out it's weird that a "Shape" is returned from calling strides.

Reply:
It's not as weird with `ttnn::SimpleShape` since this now just represents an arbitrary N-rank vector. But things like `.volume()` would be weird, I agree.

Reply:
Agreed with both. We will improve this in one of the next PRs. 
This change definitely makes it better